src/parser/Expression.ts

```diff
@@ -225,14 +225,6 @@ export class FunctionExpression extends Expression implements TypedefProvider {
         this.body = options.body;
         this.returnTypeExpression = options.returnTypeExpression;
 
-        //if there's a body, and it doesn't have a SymbolTable, assign one
-        if (this.body) {
-            if (!this.body.symbolTable) {
-                this.body.symbolTable = new SymbolTable(`Block`, () => this.getSymbolTable());
-            } else {
-                this.body.symbolTable.pushParentProvider(() => this.getSymbolTable());
-            }
-        }
```

Comment:
So for anonymous functions, they don't have a parent FunctionStatement. I don't think we can eliminate the symbolTable in here. For example:

```brightscript
sub test()
    doSomething = sub()
        'i have no immediate parent FunctionStatement
    end sub
end sub
```

Reply:
Nevermind. I misunderstood what was happening here. FunctionExpression is still getting a symbolTable, we've just eliminated the one in `Body` in this specific situation